static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
IE8 doesn't support media queries, so this calc would be ignored anyway. IE9 supports calc. You could be right about the previous one though.